Orchestrator/src/main/java/com/tainavbot/server/orchestrator/infrastructure/adapter/primary/api/dto/enumeration/HealthStatusEnumResponse.java

```diff
@@ -0,0 +1,5 @@
+package com.tainavbot.server.orchestrator.infrastructure.adapter.primary.api.dto.enumeration;
+
+public enum HealthStatusEnumResponse {
+    OK, DEAD
+}
```

Comment:
How many Enums that do the same thing do you wanna have

Reply:
up and braught down to minimum in next version

Reply:
*brought